src/wallet/wallet.cpp

```diff
@@ -1725,7 +1725,7 @@ boost::optional<uint256> CWallet::GetSproutNoteNullifier(const JSDescription &js
  */
```

Comment:
I have not reviewed `wallet.cpp` for thread safety.

Reply:
I still have not, but I don't think that this PR can introduce any additional threading problems in wallet.cpp.